singularity-opac/constants/constants.hpp

```diff
@@ -18,14 +18,6 @@
 
 #include <ports-of-call/portability.hpp>
 
-#ifdef SINGULARITY_ENABLE_EXCEPTIONS
-#include <stdexcept>
-#define CONSTANTS_ERROR(x) (throw std::runtime_error(x))
-#else
-#define CONSTANTS_ERROR(x) printf("%s", x)
-#endif
-#define UNDEFINED_ERROR CONSTANTS_ERROR("DEFINE ME\n")
```

Comment:
These are unneeded as we have a whole set of errors macros in their own header file.